tests/components/stream/test_recorder.py

```diff
@@ -39,9 +40,9 @@ async def stream_component(hass):
 
 
 @pytest.fixture
-def filename(tmpdir):
+def filename(tmp_path: Path) -> str:
     """Use this filename for the tests."""
-    return f"{tmpdir}/test.mp4"
+    return f"{tmp_path}/test.mp4"
```

Comment:
Side note: We should build the full path first, before creating the string.

Reply:
Done